wandb/sdk/interface/interface.py

```diff
@@ -397,6 +399,32 @@ def _make_artifact_manifest(
                 proto_extra.value_json = json.dumps(v)
         return proto_manifest
 
+    def publish_link_artifact(
+        self,
+        run: "Run",
+        artifact: Union[Artifact, PublicArtifact],
+        portfolio_name: str,
+        aliases: Iterable[str],
+        entity: Optional[str] = None,
+        project: Optional[str] = None,
+    ) -> None:
+        link_artifact = pb.LinkArtifactRecord()
+        if isinstance(artifact, Artifact):
+            link_artifact.client_id = artifact._client_id
+        else:
+            link_artifact.server_id = artifact.id if artifact.id else ""
```

Comment:
do you allow empty string for `server_id` in protobuf you could just not assign anything and check that this field exists, but it up to your implementation